src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Table } from '../Table/Table';
+
+export const PeopleTable = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setIsError] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
When you go to this page, you can see the UI glitch. There is unnecessary "There are no people on the server", for few milliseconds and then it hides. The solution is to set loading state to true and then just set it to false when data is loaded.

Then the UI won't glitch

Reply:
I mean, set it initially to `true`

`const [isLoading, setIsLoading] = useState<boolean>(true);`